plugins/ru.runa.gpd/src/ru/runa/gpd/search/SearchPage.java

```diff
@@ -143,4 +145,12 @@ protected void clear() {
             contentProvider.clear();
         }
     }
+    
+    private void selectFormOrFormValidation(ElementMatch elementMatch) {
```

Comment:
```suggestion
    private void selectFormNode(ElementMatch elementMatch) {
```
Мне кажется это название больше соответствует смыслу метода, т.к. он фокусируется на узле формы вне зависимости от того, как его вызвали.